### PR TITLE
make it clear we need full url, not just host

### PR DESCRIPTION
**Pull request by imnotashrimp**
# What changed

@idohalevi , take it away

----

## Pages to review

- [ ] Page1
- [ ] Page2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

**Comment by idohalevi** [comment]
@imnotashrimp FAAAASSSSSTTTT

File: _source/logzio_collections/_log-sources/guardduty.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ In the _Environment variables_ section, set your Logz.io account token, URL, and
 | Parameter | Description |
 |---|---|
 | TOKEN <span class="required-param"></span> | {% include log-shipping/replace-vars.html token='noReplace' %} <!-- logzio-inject:account-token --> |
-| URL <span class="required-param"></span> | {% include log-shipping/replace-vars.html listener='noReplace' %} <!-- logzio-inject:listener-url --> |
+| URL <span class="required-param"></span> | Protocol, listener host, and port (for example, `https://<<LISTENER-HOST>>:8071`). <br > {% include log-shipping/replace-vars.html listener='noReplace' %} <!-- logzio-inject:listener-url --> |
```
Comment (imnotashrimp) [inline _source/logzio_collections/_log-sources/guardduty.md]:
```suggestion
| URL <span class="required-param"></span> | Protocol, listener host, and port (for example, `https://<<LISTENER-HOST>>:8071`). <br > {% include log-shipping/replace-vars.html listener=true %} <!-- logzio-inject:listener-url --> |
```

File: _source/logzio_collections/_log-sources/kinesis.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ In the _Environment variables_ section, set your Logz.io account token, URL, and
 | Parameter | Description |
 |---|---|
 | TOKEN <span class="required-param"></span> | {% include log-shipping/replace-vars.html token='noReplace' %} <!-- logzio-inject:account-token --> |
-| URL <span class="required-param"></span> | {% include log-shipping/replace-vars.html listener='noReplace' %} <!-- logzio-inject:listener-url --> |
+| URL <span class="required-param"></span> | Protocol, listener host, and port (for example, `https://<<LISTENER-HOST>>:8071`). <br > {% include log-shipping/replace-vars.html listener='noReplace' %} <!-- logzio-inject:listener-url --> |
```
Comment (imnotashrimp) [inline _source/logzio_collections/_log-sources/kinesis.md]:
```suggestion
| URL <span class="required-param"></span> | Protocol, listener host, and port (for example, `https://<<LISTENER-HOST>>:8071`). <br > {% include log-shipping/replace-vars.html listener=true %} <!-- logzio-inject:listener-url --> |
```

**Comment by imnotashrimp** [comment]
@idohalevi YALLA

**Pull request merged**